docs/lake/design.md

```diff
@@ -301,9 +302,9 @@ according to configured policies and logic.
 
 Data is read from one or more pools with the `query` command.  The pool/branch names
 are specified with `from` at the beginning of the Zed query along with an optional
-time range using `range` and `to`.  The default output format is ZNG though this
-can be overridden with `-f` to specify one of the various supported output
-formats.
+time range using `range` and `to`.  The default output format is ZSON (if to a
+terminal) or ZNG (if redirected elsewhere) though this can be overridden with
```

Comment:
```suggestion
time range using `range` and `to`.  The default output format is ZSON for
terminals and ZNG otherwise, though this can be overridden with
```